src/controllers/swapAndBridge/swapAndBridge.ts

```diff
@@ -745,8 +747,57 @@ export class SwapAndBridgeController extends EventEmitter {
         ) {
           let routeToSelect
           let routeToSelectSteps
+          let routes = quoteResult.routes || []
+
+          // Exclude routes that require an additional protocol fee in a specific token
+          // if the user does not have sufficient balance of that token.
+          try {
+            routes = routes.filter((route) => {
+              let shouldFilterOut = false
+              if (!route.userTxs) return !shouldFilterOut
+
+              const bridgeTx = route.userTxs.find((tx) => tx.userTxType === 'fund-movr') as
```

Comment:
You can use the `getIsBridgeTxn`, making this `find((tx) => getIsBridgeTxn(tx.userTxType))`